spec/factories/users.rb

```diff
@@ -3,15 +3,52 @@
     email 'test@test.com'
     password '12345'
     password_confirmation '12345'
+    current_block_id ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.